.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Feature Request
+about: Suggest a new idea that you believe will help us improve the project
+title: 'FEATURE REQUEST: Title of new feature'
+labels: feature request
+---
+
+## Feature Request
+
+### Description
+
+Describe the feature
+
+### Benefit / Value
+
+Describe what problem this feature aims to solve or what benefits can come out from this feature
+
+### Proposal
+
+State the proposed solution
+
+### Alternative solutions
+
+State other solutions you
```

Comment:
the phrase is incomplete.
`State other solutions you might have to propose` (?)